readme.md

```diff
@@ -130,4 +86,42 @@ Specifying `--js` tells the generator to use JavaScript.
 
 >**Note:** Do not use this flag when you pass `react` as framework argument.
 
+## Running the Generated Site
+
+Office Add-ins must be hosted in an HTTPS site. Yo Office generates a [Browsersync](https://browsersync.io/) configuration file (`bsconfig.json`) that references a self-signed certificate. Your computer will need to trust the certificate before you can use the generated add-in. 
+
+> **Important:** Before you continue with this Readme, take the following steps (Windows):

```

Comment:
(See note below. Consider removing this note if you update the "Adding Self-Signed Certificates as Trusted Root Certificate" page with correct/complete guidance.)